qt/ts/Makefile

```diff
@@ -23,7 +23,8 @@ all: check
 PHONY: build
 build:
 	$(BIN)/tsc --build
-	$(BIN)/sass --no-source-map scss:../aqt_data/web
+	$(BIN)/sass --no-source-map scss:css
+	rsync -a css ../aqt_data/web
```

Comment:
Is that correct? IIRC that will create a css folder inside the web folder.

Do you have a backup of an earlier lockfile to see which package change broke the behaviour?

Reply:
You are right. It broke. I have no lock file as I just did a clean install with `git clean -xfd .` (to ensure it did not have broke). I accidentally closed this.